vl-convert-python/vl_convert.pyi

```diff
@@ -0,0 +1,801 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Any, Literal, TypeAlias
+
+    FormatLocaleName: TypeAlias = Literal[

```

Comment:
`FormatLocaleName` & `TimeFormatLocaleName` I got by following the links that you had in the docs.

The repos seem to have remained pretty constant, so maintaining these (hypothetically) would be on the scale of 2-7 years between changes.